File: BCW5/PHP/lcm.php
<?php 
    function gcd($a, $b) {
        if ( $b == 0 ) {
            return $a;
        }
        return gcd($b, $a%$b);
    }


    $file = fopen("Task.in", "r");
    $file1 = fopen("Task.out", "w");

    $str = fgets($file);

    $numbers = explode(' ', $str);


    $a = $numbers[0] / gcd($numbers[0], $numbers[1]) * $numbers[1];

    fwrite($file1, $a);

    fclose($file);
    fclose($file1);
?>

File: BC - w2/lcm.c
#include <stdio.h>

int gcd (int a, int b) {
    if ( b == 0 ){
        return a;
    }
        return gcd( b, a%b );
}


int main(){
        FILE *in = fopen("task.in", "r");
        FILE *out = fopen("task.out", "w");

        int a, b;

        fscanf(in, "%d %d", &a, &b);

        a = a/gcd(a, b) * b;
        fprintf(out, "%d\n", a);

        fclose(in);
        fclose(out);
        return 0;
}
